src/index.html

```diff
@@ -13,6 +13,38 @@
     />
```

Comment:
You should link to the CSS file, not the SCSS file. SCSS files are preprocessed to produce CSS that browsers can understand.